app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbooks(playbooks_names)
+      import_playbooks_content playbooks(playbooks_names)
+    end
+
+    def import_playbooks_content(playbooks)
+      params = { changed: { update: [], create: [] } }
+      playbooks.each do |playbook|
+        new_playbook = parse_playbook playbook
+        job_template_id = JobTemplate.where(name: new_playbook[:name]).pluck(:id).first
```

Comment:
There is a new method in Rails 6 for this:
```suggestion
        job_template_id = JobTemplate.where(name: new_playbook[:name]).pick(:id)
```